drivers/spi/spi_andes_atcspi200.c

```diff
@@ -0,0 +1,629 @@
+/*
+ * Copyright (c) 2019 Andes Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#define LOG_LEVEL CONFIG_SPI_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(spi_atcspi200);
+
+#include "spi_context.h"
+#include <zephyr/device.h>
+#include <zephyr/drivers/spi.h>
+
+#define DT_DRV_COMPAT andestech_atcspi200
+
+#define REG_IDR         0x00
+#define REG_TFMAT       0x10
+#define REG_DIRIO       0x14
+#define REG_TCTRL       0x20
+#define REG_CMD         0x24
+#define REG_ADDR        0x28
+#define REG_DATA        0x2c
+#define REG_CTRL        0x30
+#define REG_STAT        0x34
+#define REG_INTEN       0x38
+#define REG_INTST       0x3c
+#define REG_TIMIN       0x40
+#define REG_MCTRL       0x50
+#define REG_SLVST       0x60
+#define REG_SDCNT       0x64
+#define REG_CONFIG      0x7c
+
+#define DEV_CFG(dev) ((const struct spi_atcspi200_cfg * const) (dev)->config)
+#define DEV_DATA(dev) ((struct spi_atcspi200_data *) (dev)->data)
```

Comment:
We are removing `DEF_CFG` and `DEV_DATA` from drivers. Do not use it.

Reply:
Thanks for reviewing.

I have remove the inappropriate usage.